.github/workflows/metrics_weekly_sendgrid.yml

```diff
@@ -32,3 +32,20 @@ jobs:
           GOOGLE_API_CREDS: ${{ secrets.GOOGLE_API_CREDS }}
         run: |
           echo "$(source venv/bin/activate && python examples/metrics.py --period weekly --org sendgrid)"
+  notify-on-failure:
```

Comment:
I am not sure just this will work because the problem is that the workflow runs themselves do not fail if there is an issue. [Here](https://github.com/sendgrid/dx-automator/runs/3884252858?check_suite_focus=true) is a failed run for reference

Reply:
Ah I see... I'll make a few changes to the workflow.